Runtime/Client/LootLockerHttpRequestData.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
I might be mistaken, but is Linq necessary?

Reply:
Correct